packages/bootstrap/scss/action-sheet/_variables.scss

```diff
@@ -72,6 +72,7 @@ $kendo-actionsheet-item-disabled-shadow: null !default;
 
 
 // Adaptive Actionsheet
+$kendo-adaptive-actionsheet-font-size: $kendo-font-size-lg !default;
```

Comment:
Is this change intended by design ?

Reply:
I have discussed this topic with @dmanova: the default font size in adaptive mode is 16px.